AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -423,15 +423,33 @@ extension CardViewController {
     private func handleCardScanningResult(_ result: Result<CardScannerCardDetails, Error>) {
         switch result {
         case let .success((number, expiryDate)):
-            items.numberContainerItem.setCardNumber(number ?? "")
+            if let number {
```

Comment:
This is behavior change, previous code **always** updates the text field, new code does it only when the number is  not nil